pkg/shell/shell.go

```diff
@@ -62,11 +63,14 @@ func setupShell(fds [3]*os.File, p Paths, spawn bool) (*eval.Evaler, func()) {
 	}
 }
 
-func evalInTTY(ev *eval.Evaler, fds [3]*os.File, src parse.Source) error {
+func evalInTTY(ev *eval.Evaler, fds [3]*os.File, src parse.Source) (float64, error) {
```

Comment:
I don't see a good reason that `evalInTTY` should return the duration; this can easily be done by the caller:

```go
start := time.Now()
err := evalInTTY(...)
end := time.Now()
duration := end.Sub(start)
```

Reply:
That's reasonable, but note that the reason I did it this way is there are three callers of `evalInTTY` and two of them care about the duration. The third, in *pkg/shell/script.go*, might want to expose the duration in the future. It seems redundant, and risky, to have those callers all compute the duration in potentially incompatible fashion. Do you still want me to move the duration calculation to each call site?